packages/react-components/review/stable/react-components.api.md

```diff
@@ -677,6 +685,23 @@ export interface GridLayoutStyles extends BaseCustomStyles {
     children?: IStyle;
 }
 
+// @beta
```

Comment:
Do we allow beta exports in the stable API files?

Reply:
(I know it's not in the communication-react API file but wanted to check if the API files should be consistent with the public api file while also have internal exports)